sdk/communication/azure-communication-phonenumbers/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 1.2.0 (Unreleased)
 
+- Added support for proactive refreshing of tokens
+  - `CommunicationTokenCredential` exposes a new boolean keyword argument `refresh_proactively` that defaults to `False`. If set to `True`, the refreshing of the token will be scheduled in the background ensuring continuous authentication state.
```

Comment:
I'm not sure this needs to be in every package changelog. Most packages weren't exposing these types in the public API - I think it was only Chat and Identity.

Reply:
actually, only chat